deploy/fab/dropq_environment.yml

```diff
@@ -4,7 +4,7 @@ dependencies:
 - pandas>=0.20.1
 - flask
 - greenlet
-- bokeh
+- bokeh>=0.12.7
```

Comment:
This needs to be set in `deploy/fab/drop_environment.yml`

Reply:
This is `deploy/fab/drop_environment.yml` 🙂

Reply:
Oops!